### PR TITLE
pin

### PR DESCRIPTION
**Pull request by DianeJing**

### CODE REVIEW

**Comment by impappl** [comment]
Congratulations on your first pull request 👍 🎉

**Comment by zsu678** [comment]
good job

File: _pins/DianeJing.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+...
+githubangle:DiangeJing
```
Comment (impappl) [inline _pins/DianeJing.json]: This is a typo! Try 

`githubHandle: DianeJing`

**Comment by hollenberry** [comment]
@DianeJing It looks like you just need to do a few more things and you'll be all set to merge! You'll need to set up dashes instead of dots, insert some spaces, and camel case githubHandle. 

If you need any help, please check out the following video: https://vimeo.com/201011288 

Or let us know if you need anything extra!

**Comment by crichID** [comment]
@DianeJing I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6688/commits/5f29bbe0fd27b40be0546d760589eb8ceb2a19f2. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**